api/lib/api/performance_manager/vehicle_event_metrics.ex

```diff
@@ -0,0 +1,13 @@
+defmodule Api.PerformanceManager.VehicleEventMetrics do
+  use Ecto.Schema
+
+  @primary_key {:trip_stop_hash, :binary_id, autogenerate: true}
```

Comment:
This primary key is not autogenerated. Not sure if that's required per Ecto, but  we don't want these being auto inserted if values are provided for all of the other schema fields. 

I'm not sure totally sure of the difference in ecto between `:binary` and `:binary_id` but `:binary, autogenerate: false` could be more accurate to our `sqlachemy` schema here.

Reply:
i changed this to false.

ecto inserts an `id` field if you don't define a primary key, causing schema mismatches. for primary keys, they support `:id` and `:binary_id` types.

Reply:
You can also do `@primary_key false` if there's no primary key for the table.